bin/common/install.py

```diff
@@ -75,10 +75,10 @@ def hp_repo():
     x("""cat > /etc/yum.repos.d/hp.repo << EOF
 [HP-Proliant]
 name=Software Delivery Repository \$releasever - \$basearch
-baseurl=http://downloads.linux.hpe.com/SDR/repo/spp/rhel/\$releasever/\$basearch/current/
+baseurl=http://downloads.linux.hp.com/SDR/downloads/ServicePackforProLiant/RedHat/$releasever/$basearch/current/
```

Comment:
I think we should use https when possible, this URL seems to be the official one on HTTPS:
https://downloads.linux.hpe.com/SDR/downloads/ServicePackforProLiant/RedHat/$releasever/$basearch/current/